### PR TITLE
Add alphaFeature tag to TTLAfterFinished e2e test

### PR DESCRIPTION
**Pull request by zhuangqh**
Signed-off-by: zhuangqh <zhuangqhc@gmail.com>

**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

For the convenience of focusing or skipping all of the alpha stage feature in e2e test.
like
```
  make test-e2e-node \
    SKIP="\[NodeAlphaFeature:.+\]"
```

What's more, e2e-node-alpha test [here](https://github.com/kubernetes/test-infra/blob/d810afc5119bf5f6e4642ee1e878a56160770e3c/config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml#L227), focusing on '[NodeAlphaFeature:.+\]',  would miss this case.

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

PTAL @janetkuo 

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by zhuangqh** [comment]
> Thanks for your pull request. Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> 📝 **Please follow instructions at https://git.k8s.io/community/CLA.md#the-contributor-license-agreement to sign the CLA.**
> 
> It may take a couple minutes for the CLA signature to be fully registered; after that, please reply here with a new comment and we'll verify. Thanks.
> 
> * If you've already signed a CLA, it's possible we don't have your GitHub username or you're using a different email address.  Check your existing CLA data and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> * If you signed the CLA as a corporation, please sign in with your organization's credentials at https://identity.linuxfoundation.org/projects/cncf to be authorized.
> * If you have done the above and are still having issues with the CLA being reported as unsigned, please email the CNCF helpdesk: [helpdesk@rt.linuxfoundation.org](mailto:helpdesk@rt.linuxfoundation.org)

done

File: test/e2e/common/ttlafterfinished.go [hunk 1]
```diff
@@ -32,11 +31,10 @@ import (
 
 const dummyFinalizer = "k8s.io/dummy-finalizer"
 
-var _ = framework.KubeDescribe("TTLAfterFinished", func() {
+var _ = framework.KubeDescribe("[Feature:TTLAfterFinished][NodeAlphaFeature:TTLAfterFinished]", func() {
```
Comment (fejta) [inline test/e2e/common/ttlafterfinished.go]: Please move this test to test/e2e/node

Comment (zhuangqh) [reply test/e2e/common/ttlafterfinished.go]: done @fejta 😄

**Review by fejta — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**